pages/guide.md

```diff
@@ -100,6 +101,16 @@ Note that while connections to port 80 are insecure, even for redirects, the use
 
 HSTS mitigates the security impact of connections over port 80, while allowing agencies the flexibility to continue redirecting legacy clients or clients which have not yet received an HSTS policy for the target domain.
 
+### If I turn off plain HTTP on my server entirely, do I still need HSTS?
+
+**Yes.** Turning off HTTP support is not sufficient to prevent attacks that downgrade web browsers to plain HTTP.
```

Comment:
Consider something that addresses the case when a device only supports HTTPS (i.e., there's nothing to "turn off"):

> "Turning off HTTP support _(or only supporting HTTPS)_ is not sufficient...

When this lands, I'm likely to link to this answer in the BOD 18-01 [FAQ item](https://cyber.dhs.gov/bod/18-01/#my-device-is-https-already-and-doesnt-natively-support-hsts-can-dhs-mark-this-a-false-positive) related to this.

Reply:
@h-m-f-t Good point. I just added a commit which rephrases this to `disabled` instead of `turned off`.